sound/soc/sof/ipc.c

```diff
@@ -313,7 +313,8 @@ int snd_sof_ipc_reply(struct snd_sof_dev *sdev, u32 msg_id)
 	struct snd_sof_ipc_msg *msg = &sdev->ipc->msg;
 
 	if (msg->ipc_complete) {
-		dev_err(sdev->dev, "error: no reply expected, received 0x%x",
+		dev_dbg(sdev->dev,
+			"no reply expected, received 0x%x, will be ignored",
```

Comment:
you return -EINVAL so how is it not an error?

Reply:
Good point. I should just change this to a void function. we dont really care about the return value anyway